src/openxr/OpenXRApi.cpp

```diff
@@ -622,7 +625,7 @@ OpenXRApi::OpenXRApi() {
 	// We grab the first applicable one we find, OpenXR sorts these from best to worst choice..
 
 	Godot::print("OpenXR Swapchain Formats");
-	for (int i = 0; i < swapchainFormatCount && swapchainFormatToUse == 0; i++) {
+	for (uint i = 0; i < swapchainFormatCount && swapchainFormatToUse == 0; i++) {
```

Comment:
can we consistently use `uint32_t` for these?

Reply:
I got warning on Windows when using uint32_t that they were to small I think.. would need to look at that :)